rllib/evaluation/sampler.py

```diff
@@ -471,6 +523,8 @@ def _env_runner(
     def get_batch_builder():
         if batch_builder_pool:
             return batch_builder_pool.pop()
+        elif _use_trajectory_view_api:
```

Comment:
No pool needed. SampleCollector never gets recycled, pre-allocated buffers get reused over and over again.

Reply:
Nice!